### PR TITLE
ACCUMULO-4189 - Created User Manual and Examples pages for direct lin…

### PR DESCRIPTION
**Pull request by mikewalch**
…ks from Accumulo README

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -26,3 +26,6 @@ defaults:
       type: "pages"
     values:
       layout: "skeleton"
+
+whitelist: [jekyll-redirect-from]
+gems: [jekyll-redirect-from]
```
Comment (ctubbsii) [inline _config.yml]: Why are these lines necessary in the `_config.yml`, but not in the `_config-asf.yml`?

File: examples/latest.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+title: Accumulo Examples
+redirect_to: https://accumulo.apache.org/1.7/examples/
```
Comment (ctubbsii) [inline examples/latest.md]: Should these be relative paths instead?

File: user-manual/latest.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+title: Latest User Manual
+redirect_to: https://accumulo.apache.org/1.7/accumulo_user_manual.html
+---
+
+This page redirects to latest Accumulo Manual
```
Comment (ctubbsii) [inline user-manual/latest.md]: If I understand the redirect plugin correctly, this uses meta-refresh. The text of the page should be useful, like a link to the page, if redirects are disabled in the user's browser, or the redirect is interrupted.

File: Gemfile [hunk 4]
```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
```
Comment (ctubbsii) [inline Gemfile]: Rebase to include upstream Gemfile changes.

File: examples/latest.md (hunk 2 above)
Comment (mikewalch) [inline examples/latest.md]: Relative paths would be better but redirect_to doesn't prepend the site baseurl like redirect_from.  If I make it relative, it will be broken if you have baseurl when you run the site.

File: _config.yml (hunk 1 above)
Comment (mikewalch) [inline _config.yml]: Added in 3c4a670c67a2
